rust/pattern/variable/mod.rs

```diff
@@ -50,25 +57,49 @@ pub enum Variable {
     Concept(ConceptVariable),
     Thing(ThingVariable),
     Type(TypeVariable),
-    Unbound(UnboundVariable),
+    Value(ValueVariable),
+    Unbound(UnboundConceptVariable),
```

Comment:
Do we really need this rename then if an `Unbound` variable is only ever a `Concept`?

Reply:
Discussed in person.